compiler/rustc_codegen_ssa/src/errors.rs

```diff
@@ -1092,3 +1092,14 @@ impl<G: EmissionGuarantee> Diagnostic<'_, G> for TargetFeatureDisableOrEnable<'_
         diag
     }
 }
+
+#[derive(LintDiagnostic)]
+#[diag(codegen_ssa_mixed_export_name_and_no_mangle)]
+pub(crate) struct BuiltinMixedExportNameAndNoMangle {
```

Comment:
We are no longer in the `builtin.rs` file, let's remove the prefix.
```suggestion
pub(crate) struct MixedExportNameAndNoMangle {
```